hw/ip/rv_core_ibex/doc/_index.md

```diff
@@ -69,6 +70,34 @@ Software should take care not to enable the EDN until the entropy complex config
 When the entropy complex configuration is changed reading {{< regref "RND_DATA" >}} when it is valid will suffice to flush any old random data to trigger a new request under the new configuration.
 If a EDN request is pending when the entropy complex configuration is changed ({{< regref "RND_STATUS.RND_DATA_VALID" >}} is clear), it is advisable to wait until it is complete and then flush out the data to ensure the fresh value was produced under the new configuration.
 
+## Crash Dump Collection
+
+In general, when the CPU encounters an error, it is software's responsibility to collect error status and supply it for debug.
+
+However, there are situations where it may not be possible for software to collect any error logging.
+These situations include but are not limited to:
+* A hung transaction that causes watchdog to expire.
+* An alert escalation that directly resets the system without any software intervention.
+
+Under these situations, the software has no hints as to where the error occurred.
+To mitigate this issue, Ibex provides crash dump information that can be directly captured in the `rstmgr` for last resort debug after the reset event.
+
+The Ibex crash dump state includes the following information:
+* The current PC
+* The next PC
```

Comment:
Ah... the next PC must be used for instruction emulation, since we will want to resume past the current PC.

Reply:
ah.. @GregAC can probably better comment on that.
Since this is a crash dump, i don't actually think we intend to let software view this and jump back.

I actually just treat it as a way to see if the next PC even made sense given where the crash happened.